qpext/cmd/qpext/main.go

```diff
@@ -156,8 +247,17 @@ func (sc *ScrapeConfigurations) handleStats(w http.ResponseWriter, r *http.Reque
 	var queueProxyCancel, appCancel context.CancelFunc
 
 	defer func() {
+		if queueProxy != nil {
```

Comment:
after the pod has been up for a while, I will see quite a few context cancellation errors and im not sure why - if you see anything with the way im using cancel/close let me know. so far, i dont see why its happening

Reply:
actually I think this is just happened with the current master version of qpext - will monitor again to confirm. I think these changes /should/ fix that.